test/api_callable.js

```diff
@@ -1037,6 +1037,70 @@ describe('longrunning', function() {
           done(error);
         });
       });
+
+      it('does not return a promise when given a callback.', function(done) {
+        var func = function(argument, metadata, options, callback) {
+          callback(null, PENDING_OP);
+        };
+        var client = mockOperationsClient();
+        var apiCall = createLongrunningCall(func, client);
+        apiCall().then(function(responses) {
+          var operation = responses[0];
+          expect(operation.getOperation(
+            function(err, result, metadata, rawResponse) {
+              if (err) {
+                done(err);
+              }
+              expect(result).to.deep.eq(RESPONSE_VAL);
+              expect(metadata).to.deep.eq(METADATA_VAL);
+              expect(rawResponse).to.deep.eq(SUCCESSFUL_OP);
+              expect(client.getOperation.callCount).to.eq(1);
```

Comment:
Should we call `done()` here instaed of line 1059? Those expectations will be checked after `done` is called.